system-int-test/test/cmr/system_int_test/ingest/generics_test.clj

```diff
@@ -26,17 +26,24 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 ;; Check the functions which validate generic metadata
+
+(deftest disabled-for-ingest?
+   (testing
+    "Test function for disabling ingest of specific concepts"
+    (with-redefs [config/generic-ingest-disabled-list (fn [] [:grid])]
+     (is (= :grid (gdocs/disabled-for-ingest? :grid)) "Grid is disabled in test and will be returned")
+     (is (= nil (gdocs/disabled-for-ingest? :order-option)) "order option is NOT disabled in test and returns nil"))))
+
 (deftest validate-json-test
   (testing
    "Test that approved-generic? approves configured Generic document types"
-    (with-redefs [config/approved-pipeline-documents (fn [] {:grids ["1.0.0"]})]
+    (with-redefs [config/approved-pipeline-documents (fn [] {:grids ["1.0.0"]}) config/generic-ingest-disabled-list (fn [] [])]
       (is (true? (gcfg/approved-generic? :grids "1.0.0")) "Grids should be an approved format")
       (is (not (gcfg/approved-generic? :grid "0.0.1")) "CMR is using default configuration")
       (is (not (gcfg/approved-generic? :fake "a.b.c")) "A fake type was incorectly approved")))
-
   (testing
    "Verify that a document can be validated as approved schema."
-    (with-redefs [config/approved-pipeline-documents (fn [] {:grid ["0.0.1"]})]
+    (with-redefs [config/approved-pipeline-documents (fn [] {:grid ["0.0.1"]}) config/generic-ingest-disabled-list (fn [] [])]
```

Comment:
Can you remove the config/generic-ingest-disabled-list (fn [] [])].  Will the code still work and just use the default?

Reply:
Yea It will. I just figured it was better to put it into that test since it is being read in the actual function. Will remove from test

Reply:
you have it in other tests, so I thought that one without would be OK.